### PR TITLE
Harness test for JIRA ticket #10021

### PR DESCRIPTION
**Pull request by sylvestor88**

### CODE REVIEW

File: test/validation/org/apache/cassandra/modules/UpdateLoss.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+                double f = 0;
+                for (Row row : results) {
+                    f = row.getDouble("f");
+                }
+
+                if (f != f2)
+                {
+                    lossCount++;
+                    System.out.println("Lost Update Data: Expected " + f2 + ", Got " + f);
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/UpdateLoss.java]: You should log this, not print to console.

File: test/validation/org/apache/cassandra/modules/UpdateLoss.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import org.apache.cassandra.HarnessContext;
+import org.apache.cassandra.concurrent.DebuggableThreadPoolExecutor;
+import org.apache.cassandra.htest.Config;
+
+public class UpdateLoss extends Module
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/UpdateLoss.java]: This class and file name should be `UpdateLossModule`

File: test/validation/org/apache/cassandra/modules/UpdateLoss.java (hunk 1 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/UpdateLoss.java]: Is that log or print to console required though? I just put that as a reference. Eventually test determines if there is any loss of update.

File: test/validation/org/apache/cassandra/modules/UpdateLoss.java (hunk 2 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/UpdateLoss.java]: Changed that

**Comment by sylvestor88** [comment]
The test fails on C\* version 2.1.6 for both smaller and larger updates

File: test/validation/org/apache/cassandra/modules/UpdateLoss.java (hunk 1 above)
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/UpdateLoss.java]: No, not really.

**Pull request closed without merging**